### PR TITLE
Tesseron - Corrections of Invoice und Basic Data Connector

### PR DESCRIPTION
**Pull request by developmentLL**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json [hunk 1]
```diff
@@ -20,321 +20,6 @@
   ],
   "produces": [],
   "paths": {
-    "/ApiEnterprise/SearchEnterprise": {
```
Comment (vmanoharas) [inline certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:23]: Can you please confirm that the existing flows will not break the flows? 

The operation has been deleted. This change breaks existing flows. ValuePath: operation/SearchEnterprise.

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json [hunk 2]
```diff
@@ -20,321 +20,6 @@
...
-              },
-              "required": [
-                "referenceNumber"
-              ]
-            }
-          }
-        ]
-      }
-    },
-    "/ApiEnterprise/GetEnterprise": {
```
Comment (vmanoharas) [inline certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:103]: Can you please confirm that the existing flows will not break the flows?

The operation has been deleted. This change breaks existing flows. ValuePath: operation/GetEnterprise

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json [hunk 3]
```diff
@@ -20,321 +20,6 @@
...
-              },
-              "required": [
-                "enterpriseId"
-              ]
-            }
-          }
-        ]
-      }
-    },
-    "/ApiEnterprise/SearchEnterpriseByParameter": {
```
Comment (vmanoharas) [inline certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:201]: Can you please confirm that the existing flows will not break the flows?

The operation has been deleted. This change breaks existing flows. ValuePath: operation/SearchEnterpriseByParameter

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json [hunk 4]
```diff
@@ -888,7 +573,7 @@
               },
               "required": [
                 "BusinessPartnerId",
-                "EnterpriseName",
+                "EnterpriseName1",
```
Comment (vmanoharas) [inline certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:576]: The required input field has been added. Existing flows will not send this new field. EnterpriseName1. Kindly confirm.

**Review by vmanoharas — COMMENTED** [review]
Hello @developmentLL, 

Kindly address review comments along with below swagger error messages. Please confirm that the existing flow will not be broken. 

##[error]The input field has been changed from optional to required. ValuePath: operation/UpdateEnterprise/parameter/body/schema/SearchName/required Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/UpdateEnterprise/parameter/body/schema/EnterpriseName1 Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/CreateContact/parameter/body/schema/surName Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/UpdateContact/parameter/body/schema/surName Source: NewSwagger

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json (hunk 1 above)
Comment (developmentLL) [reply certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:23]: We will inform our customers, that this endpoint is no longer supported

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json (hunk 3 above)
Comment (developmentLL) [reply certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:201]: We will inform our customers, that this endpoint is no longer supported

File: certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json (hunk 4 above)
Comment (developmentLL) [reply certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json:576]: The previous label was not adequate. For this reason the connector wasn't able to be userd.

**Comment by vmanoharas** [comment]
> Hello @developmentLL,
> 
> Kindly address review comments along with below swagger error messages. Please confirm that the existing flow will not be broken.
> 
> ##[error]The input field has been changed from optional to required. ValuePath: operation/UpdateEnterprise/parameter/body/schema/SearchName/required Source: NewSwagger ##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/UpdateEnterprise/parameter/body/schema/EnterpriseName1 Source: NewSwagger ##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/CreateContact/parameter/body/schema/surName Source: NewSwagger ##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/UpdateContact/parameter/body/schema/surName Source: NewSwagger

Hello @developmentLL, 

Thank you very much for addressing review comments.  Could you please review above errors and confirm that the existing flows will not be broken?

**Comment by developmentLL** [comment]
Hello @vmanoharas ,
the errors will not affect the current flow. Our customers have to vcreate a new flow, becauste the old connector isnt working.

**Comment by vmanoharas** [comment]
Hello @developmentLL, 

Thank you very much for confirming.

**Review by vmanoharas — APPROVED** [review]
Dear Partner, 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fblog%2Fannouncing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins%2F&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390444301%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5MYg0GlCsic2JqzwJU0h7mIOc7uHOeL7ZWSbgq%2FFkWk%3D&reserved=0)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
4.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0)
5.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
6.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team

**Pull request merged**